src/AlohaKit.Gallery/AlohaKit.Gallery.csproj

```diff
@@ -24,9 +24,9 @@
 		<!-- Required for C# Hot Reload -->
 		<UseInterpreter Condition="'$(Configuration)' == 'Debug'">True</UseInterpreter>
 
-		<SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-ios'">14.2</SupportedOSPlatformVersion>
-		<SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-maccatalyst'">14.0</SupportedOSPlatformVersion>
-		<SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-android'">21.0</SupportedOSPlatformVersion>
```

Comment:
This should be made version agnostic as is now even part of the MAUI templates

```
$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'ios'
```

These properties should also be moved to a Directory.Build.props in the root of the repo. This way it only needs to be specified once instead of in the Gallery and in the library.